front/app/modules/commercial/content_builder/admin/components/ContentBuilderFrame/index.tsx

```diff
@@ -30,6 +30,7 @@ const ContentBuilderFrame = ({ projectId }: { projectId: string }) => {
   return (
     <Frame>
       <Element
+        id="content-builder-frame"
```

Comment:
We should indicate that these are used for e2e tests by adding e2e to the ids - `e2e-content-builder-frame`. That way, it's more clear why the ids are there.